src/python/pants/base/hash_utils.py

```diff
@@ -6,13 +6,22 @@
 
 import hashlib
 import json
-from builtins import object, open
+from builtins import bytes, object, open, str
 
+from future.moves import collections
 from future.utils import PY3
 
+from pants.base.deprecated import deprecated
 from pants.util.strutil import ensure_binary
 
 
+if PY3:
```

Comment:
Erm. I was hoping you'd just focus the TODO with an issue tied to the python 3 port to add this once we were on python 3. Having different functionality in 2 and 3 seems like a thing we want to stay away from.

Reply:
That sounds extremely reasonable, I'll make an issue and remove this.

Reply:
Removed!

Reply:
(made into #6513)